pom.xml

```diff
@@ -167,7 +169,9 @@
         <profile>
             <id>quick</id>
             <properties>
+                <!-- Allow for test-compile stage to build common-tests jar -->
                 <skipTests>true</skipTests>
+                <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
skip doesnt seem to skip javadoc compile, please fix thaaanks

Reply:
Skip does skip, my mistake